File: Proiect/src/main/java/model/Ticket.java
package model;

import java.io.Serializable;

public class Ticket implements Identifiable<Integer>, Serializable {
    private Integer ID;
    private Integer seat;
    private Long price;
    private Integer matchID;

    public Ticket(Integer seat, Long price, Integer matchID) {
        this.seat = seat;
        this.price = price;
        this.matchID = matchID;
    }

    public Integer getSeat() {
        return seat;
    }

    public void setSeat(Integer seat) {
        this.seat = seat;
    }

    public Long getPrice() {
        return price;
    }

    public void setPrice(Long price) {
        this.price = price;
    }

    public Integer getMatchID() {
        return matchID;
    }

    public void setMatchID(Integer matchID) {
        this.matchID = matchID;
    }

    @Override
    public String toString() {
        return "Ticket{" +
                "ID=" + ID +
                ", seat=" + seat +
                ", price=" + price +
                ", matchID=" + matchID +
                '}';
    }

    @Override
    public Integer getID() {
        return ID;
    }

    @Override
    public void setId(Integer id) {
        ID = id;
    }
}

File: ProiectCsharp/MPP/MPP/model/Ticket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MPP.model
{
    internal class Ticket : Identifiable<int>
    {
        private int _id;
        private int _seat;
        private long _price;
        private int _matchId;

        public Ticket(int seat, long price, int matchId)
        {
            Seat = seat;
            Price = price;
            MatchId = matchId;
        }

        public Ticket(int id, int seat, long price, int matchId)
        {
            Id = id;
            Seat = seat;
            Price = price;
            MatchId = matchId;
        }

        public int Id
        {
            get { return _id; }
            set { _id = value; }
        }

        public int Seat
        {
            get { return _seat; }
            set { _seat = value; }

        }
        public long Price
        {
            get { return _price; }
            set { _price = value; }
        }
        public int MatchId
        {
            get { return _matchId; }
            set { _matchId = value; }
        }
    }
}
